chkcryptoboot_hook

```diff
@@ -3,6 +3,7 @@
 check_mbr ()
 {
     if [ -n "$BOOTDISK" -a -s /etc/chkcryptoboot/disk-head.sha512 ]; then
+	resolve_device "$BOOTDISK"
```

Comment:
For what reason is this resolve_device being called for? We don't need to resolve the actual device we're booting from.

Reply:
This is being called in order to resolve #10

Reply:
This is the BOOTDISK, not the ESP partition. Also, you can't even use UUID with this, since this is supposed to be the path to the whole disk, not a partition. The only possible id is the PTUUID, which resolve_device doesn't support.